app/views/players/_form.html.slim

```diff
@@ -40,3 +40,11 @@
 			.form-group
 				label.d-block for="player_manual_seed" Manual Seed
 				=> f.text_field :manual_seed, placeholder: 'Set seed', class: 'manual_seed form-control'
+	- if defined?(new_registration) and not defined?(organiser_view)
+		= f.input :consent_data_sharing, as: :boolean,
+						class: 'form-check-input', input_html: { class: 'mr-1' },
+						inline_label: 'I consent for my personal data to be shared on this website'
+		- if @tournament.nrdb_deck_registration?
+			= f.input :consent_deck_sharing_with_to, as: :boolean,
+							class: 'form-check-input', input_html: { class: 'mr-1' },
+							inline_label: 'I consent for my decks to be shared with the organiser'
```

Comment:
What happens if you don't consent?

Reply:
It disables the button to register for the tournament.

Reply:
excellent.